drop-config/src/lib.rs

```diff
@@ -13,6 +13,11 @@ pub struct DropConfig {
     pub storage_path: String,
     // If set the checksum events will be emited for every file of this or bigger size
     pub checksum_events_size_threshold: Option<usize>,
+    // If set to 1, we will generate progress events for every chunk (256KB)
+    // If set to 2, we will generate progress events for every 2 chunk (512KB)
+    // If set to 3, we will generate progress events for every 3 chunk (768KB), and so on..
+    // The default logic uses this as 1.
+    pub checksum_events_granularity: Option<u64>,
```

Comment:
The chunk size is an implementation detail. Apps should be able to specify this number in bytes.
Additionally, I'd say this shouldn't be `Option<>` and in the config JSON structure we should use
`#[serde(default =...)]`